sdk/core/azure-json/src/main/java/com/azure/json/package-info.java

```diff
@@ -2,6 +2,499 @@
+ *      * null if it was pointing to JSON null.
+ *      * &#64;throws IOException If an error occurs while reading the FluentJsonSerializableExample.
+ *      *&#47;
+ *     public static FluentJsonSerializableExample fromJson&#40;JsonReader jsonReader&#41; throws IOException &#123;
+ *         &#47;&#47; 'readObject' will initialize reading if the JsonReader hasn't begun JSON reading and validate that the
+ *         &#47;&#47; current state of reading is a JSON start object. If the state isn't JSON start object an exception will be
+ *         &#47;&#47; thrown.
+ *         return jsonReader.readObject&#40;reader -&gt; &#123;
+ *             FluentJsonSerializableExample deserializedValue = new FluentJsonSerializableExample&#40;&#41;;
+ *
+ *             while &#40;reader.nextToken&#40;&#41; != JsonToken.END_OBJECT&#41; &#123;
+ *                 String fieldName = reader.getFieldName&#40;&#41;;
+ *                 reader.nextToken&#40;&#41;;
+ *
+ *                 &#47;&#47; In this case field names are case-sensitive but this could be replaced with 'equalsIgnoreCase' to
+ *                 &#47;&#47; make them case-insensitive.
+ *                 if &#40;&quot;int&quot;.equals&#40;fieldName&#41;&#41; &#123;
+ *                     deserializedValue.setAnInt&#40;reader.getInt&#40;&#41;&#41;;
+ *                 &#125; else if &#40;&quot;boolean&quot;.equals&#40;fieldName&#41;&#41; &#123;
+ *                     deserializedValue.setABoolean&#40;reader.getBoolean&#40;&#41;&#41;;
+ *                 &#125; else if &#40;&quot;string&quot;.equals&#40;fieldName&#41;&#41; &#123;
+ *                     deserializedValue.setAString&#40;reader.getString&#40;&#41;&#41;;
+ *                 &#125; else if &#40;&quot;decimal&quot;.equals&#40;fieldName&#41;&#41; &#123;
+ *                     &#47;&#47; For nullable primitives 'getNullable' must be used as it will return null if the current token
+ *                     &#47;&#47; is JSON null or pass the reader to the non-null callback method for reading, in this case for
+ *                     &#47;&#47; Double it is 'getDouble'.
+ *                     deserializedValue.setANullableDecimal&#40;reader.getNullable&#40;JsonReader::getDouble&#41;&#41;;
+ *                 &#125; else &#123;
+ *                     &#47;&#47; Fallthrough case of an unknown property. In this instance the value is skipped, if it's a JSON
+ *                     &#47;&#47; array or object the reader will progress until it terminated. This could also throw an exception
+ *                     &#47;&#47; if unknown properties should cause that or be read into an additional properties Map for further
+ *                     &#47;&#47; usage.
+ *                     reader.skipChildren&#40;&#41;;
+ *                 &#125;
+ *             &#125;
+ *
+ *             return deserializedValue;
+ *         &#125;&#41;;
+ *     &#125;
+ * &#125;
+ * </pre>
+ * <!-- end com.azure.json.JsonSerializable.fluent -->
```

Comment:
Somewhere you should show what the code is to call these `toJson` and `fromJson` methods, including what is returned / provided to the methods.

Reply:
That's done later on in the package-info in another section, do you want it included here earlier though?